src/util/storageOptions.tsx

```diff
@@ -1,31 +1,37 @@
+import { LxdSettings } from "types/server";
+
 export const dirDriver = "dir";
 export const btrfsDriver = "btrfs";
 export const lvmDriver = "lvm";
 export const zfsDriver = "zfs";
 export const cephDriver = "ceph";
 
-export const storageDrivers = [
-  {
-    label: "Directory",
-    value: dirDriver,
-  },
-  {
-    label: "Btrfs",
-    value: btrfsDriver,
-  },
-  {
-    label: "LVM",
-    value: lvmDriver,
-  },
-  {
-    label: "ZFS",
-    value: zfsDriver,
-  },
-  {
-    label: "Ceph",
-    value: cephDriver,
-  },
-];
+type StorageOption = {
```

Comment:
This should be `type Option = OptionHTMLAttributes<HTMLOptionElement>;`, or use the `OptionHTMLAttributes<HTMLOptionElement>` directly.

Reply:
This is really good to know, thanks! :+1: